spec/views/events/participants.html.erb_spec.rb

```diff
@@ -49,17 +49,42 @@
     expect(rendered).to have_text(t(:unnecessary, scope:'events.participants'))
   end
 
-  it "displays print badges button" do
+  it "displays print badges button (in event execution phase)" do
+    @event.published = true
+    @event.application_deadline = Date.yesterday
+    @event.application_status_locked = true
     render
     expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges', disabled: false))
   end
 
   it "disables the print badges button when there are no participants" do
+    @event = assign(:event, FactoryGirl.create(:event, :in_execution_phase))
     @participants = []
     render
     expect(rendered).to have_link(t(:print_button_label, scope: 'events.badges', disabled: true))
   end
 
+  it "does not show the print badges button when the event is in draft phase" do
+    @event = assign(:event, FactoryGirl.create(:event_with_accepted_applications, :in_draft_phase))
```

Comment:
wait, you mean like draft and selection are a subset of when they are no participants?

I think these tests should still be there, because in these cases there should be no button at all, while in the "normal" case with no participants it's disabled